master-server/src/main/java/com/replicated_log/master_server/repository/Storage.java

```diff
@@ -0,0 +1,12 @@
+package com.replicated_log.master_server.repository;
+
+import java.util.List;
+
+public interface Storage<T> {
+
+    boolean append(T item);
+
+    List<T> findAll();
+
+    List<T> getStorage();
```

Comment:
why do we need these findAll and getStorage methods with the same logic?

Reply:
removed findAll and left just getStorage method